src/components/Payments/Tabs/Swap.tsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react"
+
+import Subtab from "../../ui/Subtab";
+
+const PaymentsSwap = () => {
+    const [subtab, setSubtab] = useState(0)
+
+    return (
```

Comment:
Can we use Tab component from headlessui? The existing tabs component uses that and I would like the interface to remain the same. This makes it harder to maintain